File: Problems/No.119_Pascal's_Triangle_II/pascal's_triangle_II.py
class Solution(object):
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        
        # parameter
        ans = [1]
        
        # start compute
        for i in range(rowIndex):
            ans.append(1)
            for j in range(i, 0, -1):
                ans[j] += ans[j-1]
        
        return ans
        

File: Problems/No.119_Pascal's_Triangle_II/pascal's_triangle_II.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        
        // parameter
        vector<int> ans = {1};
        
        // start compute
        for (int i=0; i<rowIndex; i++) {

            ans.push_back(1);
            for (int j=i; j>0; j--) ans[j] += ans[j-1];
            
        }
        
        return ans;
        
    }
};
